launcher/ui/MainWindow.cpp

```diff
@@ -929,12 +244,13 @@ MainWindow::MainWindow(QWidget *parent) : QMainWindow(parent), ui(new MainWindow
     // Add the news label to the news toolbar.
     {
         m_newsChecker.reset(new NewsChecker(APPLICATION->network(), BuildConfig.NEWS_RSS_URL));
-        newsLabel = new QToolButton();
-        newsLabel->setIcon(APPLICATION->getThemedIcon("news"));
-        newsLabel->setSizePolicy(QSizePolicy::Expanding, QSizePolicy::Preferred);
-        newsLabel->setToolButtonStyle(Qt::ToolButtonTextBesideIcon);
-        newsLabel->setFocusPolicy(Qt::NoFocus);
-        ui->newsToolBar->insertWidget(ui->actionMoreNews, newsLabel);
+        newsLabel = dynamic_cast<QToolButton*>(ui->newsToolBar->widgetForAction(ui->actionNewsLabel));
+
+        //add a spacer before the more news button
```

Comment:
idk how to fix that without adding the news label toolbutton manually, i tried setting the size policy but it has no effect :/

Reply:
seems like it indeed will not be possible without manually inserting the toolbutton, since it gets added via `addAction` in the generated file, which i think forces the layout to be that way. Still, I think it is still worth it to manually add it :slightly_smiling_face: